.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Hey Gabe, can you delete this .vscode folder. I pushed it too by mistake when I opened multiple servers from VS Code for the same project...don't want unnecessary folders/files in the repo, you know. Just make sure there's nothing important in it before deleting.

Reply:
will do